lib/longtail/stackoverflow/posts.pl

```diff
@@ -75,7 +75,8 @@
                 # 1 = question, 2 = answer, 3 might be external
                 my $link_type_id = $4;
                 #warn "$id -> $post_id -> $related_post_id -> $link_type_id\n";
-                push (@{$post_links{$post_id}} ,$related_post_id);
+                $post_links{$post_id}{$related_post_id} = 1;
+                $post_links{$related_post_id}{$post_id} = 1;
```

Comment:
Yeah.  Would we possibly want the link counts in the future, e.g. to determine the most popularly linked and do something?  If so, rather than set to `1` I would `++$post_links{...` to keep count.